From 83136384582280bde3379ee87898891d88463930 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marek=20Marczykowski-G=C3=B3recki?=
 <marmarek@invisiblethingslab.com>
Date: Tue, 25 Jun 2019 19:59:17 +0200
Subject: [PATCH] Add "plymouth.ignore-serial-consoles" boot option by default

This preserve graphical boot even if plymouth detects /dev/hvc0 as a
serial console.

QubesOS/qubes-issues#3849
---
 pyanaconda/installclasses/qubes.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pyanaconda/installclasses/qubes.py b/pyanaconda/installclasses/qubes.py
index b5caff037..0ab8ce553 100644
--- a/pyanaconda/installclasses/qubes.py
+++ b/pyanaconda/installclasses/qubes.py
@@ -45,6 +45,7 @@ class InstallClass(BaseInstallClass):
     installUpdates = False
 
     bootloaderTimeoutDefault = 5
+    bootloaderExtraArgs = ["plymouth.ignore-serial-consoles"]
 
     tasks = [(N_("Minimal"), ["base", "base-x", "kde-desktop-qubes", "qubes" ]) ]
 
-- 
2.20.1

